pages/services-procurement/index.js

```diff
@@ -0,0 +1,128 @@
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import { CardContent } from '@material-ui/core';
+
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import ServicesNav from '../../components/ServicesNav'
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
Can't you move this to its own util file or something? It looks duplicated across all of the service pages.